lib/dockerize_rails/docker_commands/docker_build.rb

```diff
@@ -16,10 +21,15 @@ def self.build_rails
 
       def self.build_mysql
         if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'mysql'
+          if Docker::Image.exist? DockerHelpers.get_name(:mysql, :image)
+            puts " ==> Image >#{DockerHelpers.get_name(:mysql, :image)}< already exists".green
+            return 0
+          end
           build_docker_image(
-            "#{Constants::CONFIG_DIRECTORY_NAME}/#{Constants::MYSQL_DIRECTORY_NAME}/Dockerfile",
-            "#{DRConfig.application_name}_mysql"
+              "#{Constants::CONFIG_DIRECTORY_NAME}/#{Constants::MYSQL_DIRECTORY_NAME}/Dockerfile",
+              DockerHelpers.get_name(:mysql, :image)
```

Comment:
__[rubocop]__
* Align the parameters of a method call if they span more than one line. [:ref](https://github.com/bbatsov/ruby-style-guide#no-double-indent)

<!-- Comment by [SideCI](https://sideci.com) -->